ramp/core/core.py

```diff
@@ -100,6 +105,10 @@ def __init__(
         if self.date_start is not None and self.date_end is not None:
             self.initialize()
 
+        # Set global random seed if it is specified
+        if self.random_seed:
```

Comment:
```suggestion
        if self.random_seed is not None:
```

I prefer the defensive coding :)